docs/edgeql/statements/describe.rst

```diff
@@ -0,0 +1,194 @@
+.. _ref_eql_statements_describe:
+
+DESCRIBE
+========
+
+:eql-statement:
+
+``DESCRIBE`` -- provide human-readable description of a concept
+
+.. eql:synopsis::
+
+    DESCRIBE SCHEMA [ AS {DDL | SDL | TEXT} ] [ VERBOSE ];
+
+    DESCRIBE <concept-type> <name> [ AS {DDL | SDL | TEXT} ] [ VERBOSE ];
+
+    # where <concept-type> is one of
+
+      OBJECT
+      ANNOTATION
+      CONSTRAINT
+      FUNCTION
+      LINK
+      MODULE
+      PROPERTY
+      SCALAR TYPE
+      TYPE
+
+Description
+-----------
+
+``DESCRIBE`` generate a human-readable description of a schema object.
+
+The output of a ``DESCRIBE`` command is a :eql:type:`str` , although
+it cannot be used as an expression in queries. It is intended as a
+quick way to look at a piece of the database schema.
+
+There are three output formats to choose from:
+
+:eql:synopsis:`AS DDL`
+    Provide a valid :ref:`DDL <ref_eql_ddl>` definition.
+
+    The :ref:`DDL <ref_eql_ddl>` generated is a complete valid
+    definition of the particular schema object from the point of view
+    of the current schema.
+
+    This is the default format.
+
+:eql:synopsis:`AS SDL`
+    Provide an :ref:`SDL <ref_eql_sdl>` definition.
+
+    The :ref:`SDL <ref_eql_sdl>` generated is a complete valid
+    definition of the particular schema object from the point of view
+    of the current schema.
+
+:eql:synopsis:`AS TEXT`
+    Provide a textual definition.
+
+    The textual definition generated is similar to :ref:`SDL
+    <ref_eql_sdl>`, but it includes all the details that are inherited
+    (if any).
+
+    The :eql:synopsis:`VERBOSE` mode enables displaying additional
+    details, such as :ref:`annotations <ref_datamodel_annotations>`
+    and :ref:`constraints <ref_datamodel_constraints>`, which are
+    otherwise omitted.
+
+When the ``DESCRIBE`` command is used with the :eql:synopsis:`SCHEMA`
+the result is a definition of the entire database schema. Only the
+:eql:synopsis:`AS DDL` option is available for schema description.
+
+The ``DESCRIBE`` command can specify the type of schema object that it
+should generate the description of:
+
+:eql:synopsis:`OBJECT <name>`
+    Match any schema object with the specified *name*.
```

Comment:
except `MODULE` (and other globals that cannot be uniquely identified just by the name).

Reply:
OK